app/jobs/application_job.rb

```diff
@@ -5,15 +5,18 @@
 # Base class for all Jobs
 class ApplicationJob < ActiveJob::Base
   # Copied from ActiveJob::Exceptions, but uses debug log level.
-  def self.retry_on(exception, wait: 3.seconds, attempts: 5, queue: nil, priority: nil)
+  def self.retry_on(exception, wait: 3.seconds, attempts: 5, queue: nil, priority: nil, jitter: ActiveJob::Exceptions.const_get(:JITTER_DEFAULT))
```

Comment:
Jitter was [introduced](https://github.com/rails/rails/pull/31872) as an option here in Rails 6.1. Adding it to the monkey patch as well only so we stick with the same API.